exporter/sumologicexporter/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sumolo
 go 1.14
 
 require (
+	github.com/klauspost/pgzip v1.2.5
```

Comment:
This dependency seems to require `cgo` and we do not build with that enabled our binaries. See makefile (CGO_ENABLED=0)

Reply:
Shouldn't it fail during building/CI?

I didn't find any `import "C"` in the pgzip repo

Reply:
It is on the README page.

Reply:
Changed to the `compress/gzip`